src/pathfinding_service/cli.py

```diff
@@ -31,6 +27,12 @@
 from raiden_libs.blockchain import get_web3_provider_info
 from raiden_libs.cli import blockchain_options, common_options, setup_sentry
 
+# pylint: disable=too-many-locals
+# there were some issues with the 'thread' resolver, remove it from the options
+config.resolver = ["dnspython", "ares", "block"]  # noqa
```

Comment:
This is still wrong

Reply:
in line 30?

Reply:
Ah no, this is about the gevent monkey patching. This needs to happen first

Reply:
hmm it works if I put it there. Any idea where I should put it instead?

Reply:
It should go where it was before, right at the top of the file